modules/openxAnalyticsAdapter.js

```diff
@@ -1014,10 +1012,13 @@ function onSlotLoadedV2({ slot }) {
     adUnit.renderTime = renderTime;
   }
 
+  if (auction.adunitCodesRenderedCount === auction.adUnitCodesCount) {
+    auction.state = AUCTION_STATES.COMPLETED;
+  }
+
```

Comment:
is it part of the fixing of the bug you mentioned ?

Reply:
Yes.  This is the only fix.  I need to add unit test to cover this.